session/create_consumer.go

```diff
@@ -69,7 +69,7 @@ func (consumer *createConsumer) Consume(requestPtr interface{}) (response interf
 	issuerID := consumer.peerID
 	if request.ConsumerInfo != nil {
 		issuerID = request.ConsumerInfo.IssuerID
-		if request.ConsumerInfo.PaymentVersion == PaymentVersionV2 {
+		if request.ConsumerInfo.PaymentVersion == PaymentVersionV3 {
```

Comment:
Why it is now called `PaymentVersionV3`?

Reply:
because we've already introduced version 2 into master a while ago, since this is incompatible, we need to switch to 3